ui/webui/test/README.rst

```diff
@@ -243,3 +266,60 @@ development environment that's independent from the host::
 
     toolbox create --image quay.io/cockpit/tasks -c cockpit
     toolbox enter cockpit
+
+Web UI integration tests
```

Comment:
Integration and end to end tests don't have to be the same. Let's call these E2E and the other ones integration tests.